harness/harness.go

```diff
@@ -104,7 +119,14 @@ func (c *H) flushToParent(format string, args ...interface{}) {
 
 	c.mu.Lock()
 	defer c.mu.Unlock()
-	io.Copy(p.w, &c.output)
+	outputBufferCopy := c.output
+	outputBufferCopy.WriteTo(p.w)
+}
+
+func (c *H) publishToReporters(name string, status reporters.TestResult) {
+	if !c.hasSub {
```

Comment:
I don't think we actually want to skip this. It's possible to miss failures and output as-is.

For example, in the `docker.torcx-manifest-pkgs` tests:

https://github.com/coreos/mantle/blob/2a5b8f93cf2b2d38a303f655b7563e0c7f890f77/kola/tests/docker/torcx_manifest_pkgs.go#L142

If that MustSSH Fails, it will hit this condition of having a subtest (the `install-torcx-profile` above), and thus not be reported as failing.